src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.c

```diff
@@ -58,12 +68,12 @@ static InitChainEntry sInitChain[] = {
     ICHAIN_VEC3F_DIV1000(scale, 100, ICHAIN_STOP),
 };
 
-static Vec3f D_8089EBA0 = { 0.0f, 80.0f, 23.0f };
-static Vec3f D_8089EBAC = { 0.0f, 80.0f, 23.0f };
+static Vec3f offsetPosition1 = { 0.0f, 80.0f, 23.0f };
+static Vec3f offsetPosition2 = { 0.0f, 80.0f, 23.0f };
 
-static u8 D_8089EE40;
+static u8 D_8089EE40; // unknown flags
```

Comment:
after taking a very quick glance I think this has the purpose of playing sfx only once across all actors when there are several actors loaded

Reply:
it seems to be related to sfx yeah, but it's also doing stuff for the hookshot platform (when it does ``& 1``) that's why I was a bit confused and didn't named it

Reply:
what's non sfx related with "the `& 1`" ?

Reply:
I misremembered sorry lol, still I have no clue what's the point of this

Reply:
I think the point is that this actor usually calls `func_8002F948` to play sound effects at the location of the camera, rather than at the position the actor is located in the environment. You hear this happening here: 

https://youtu.be/8DYjPvti7TU?si=zxAKYXuPhozdVuHn&t=1088

I imagine that if the sfx was played multiple times per frame it would sound much louder.

Reply:
``D_8089EE40`` could be ``sSfxFlags`` then? I'm not super convinced by this since it looks like the struct member